src/upstream/dotbit/mod.rs

```diff
@@ -180,8 +183,10 @@ async fn fetch_connections_by_platform_identity(
 ) -> Result<TargetProcessedList, Error> {
     if _platform == &Platform::Dotbit {
         return fetch_connections_by_account_info(_platform, identity).await;
-    } else {
+    } else if _platform == &Platform::Ethereum {
```

Comment:
Rename `_platform` to `platform` since it is not unused variable anymore.

Reply:
d